toolchain/check/handle_literal.cpp

```diff
@@ -49,6 +49,20 @@ static auto MakeI32Literal(Context& context, Parse::NodeId node_id,
        .int_id = context.ints().Add(i32_val)});
 }
 
+// Forms an IntLiteral instruction with type `BigInt` for a given literal
+// integer value, which is assumed to be unsigned.
+static auto MakeBigIntLiteral(Context& context, Parse::NodeId node_id,
+                              IntId int_id) -> SemIR::InstId {
+  // TODO: `IntId`s with different bit-widths are considered different values
+  // here. Decide how we want to canonicalize these. For now this is only used
+  // by type literals, so we rely on the lexer picking some consistent rule.
```

Comment:
I think Chandler was looking at this separately, you might want to talk with him. A TODO here might be lost, because this may be more of a TODO for the numeric literal lexer.

Reply:
I couldn't find another good place for this TODO to live. I'll try to remember it once Chandler's change lands.